### PR TITLE
Add hot_water_recirculation_pump to unavailable_periods.csv

### PR DESCRIPTION
**Pull request by joseph-robertson**
## Pull Request Description

Without it, the following error is thrown for a building with a recirc pump and a, e.g., power outage defined:
```
Could not find row='hot_water_recirculation_pump' in unavailable_periods.csv
```
Add tests in `test_schedules.rb` for recirculation pump and general water use.
Change general water use to be _unaffected_ by a power outage.

## Checklist

PR Author: Check these when they're done. Not all may apply. ~~strikethrough~~ and check any that do not apply. 

PR Reviewer: Verify each has been completed.

- [ ] ~Schematron validator (`EPvalidator.xml`) has been updated~
- [x] Sample files have been added/updated (`openstudio tasks.rb update_hpxmls`)
- [x] Tests have been added/updated (e.g., `HPXMLtoOpenStudio/tests/test*.rb` and/or `workflow/tests/test*.rb`)
- [x] Documentation has been updated
- [ ] ~Changelog has been updated~
- [x] `openstudio tasks.rb update_measures` has been run
- [x] No unexpected changes to simulation results of sample files

### CODE REVIEW

File: HPXMLtoOpenStudio/resources/data/unavailable_periods.csv [hunk 1]
```diff
@@ -24,6 +24,7 @@ pool_heater,0,1
 permanent_spa_pump,0,1
 permanent_spa_heater,0,1
 hot_water_fixtures,1,1
+hot_water_recirculation_pump,1,1
 general_water_use,1,1
```
Comment (joseph-robertson) [inline HPXMLtoOpenStudio/resources/data/unavailable_periods.csv:28]: Should general_water_use really be affected by a power outage?

File: HPXMLtoOpenStudio/resources/data/unavailable_periods.csv [hunk 2]
```diff
@@ -24,6 +24,7 @@ pool_heater,0,1
 permanent_spa_pump,0,1
 permanent_spa_heater,0,1
 hot_water_fixtures,1,1
+hot_water_recirculation_pump,1,1
```
Comment (joseph-robertson) [inline HPXMLtoOpenStudio/resources/data/unavailable_periods.csv:27]: Does 0 or 1 here depend on the _type_ of recirc pump?

Comment (shorowit) [reply HPXMLtoOpenStudio/resources/data/unavailable_periods.csv:27]: Any recirculation pump requires electricity, so definitely affected by a power outage.

**Comment by shorowit** [comment]
Is there any way to add a test that would proactively identify a missing schedule in the csv?

File: HPXMLtoOpenStudio/resources/data/unavailable_periods.csv (hunk 2 above)
Comment (joseph-robertson) [reply HPXMLtoOpenStudio/resources/data/unavailable_periods.csv:27]: Right, but what about vacancy? Should we keep it simple and just say that the recirc pump is unavailable when building is vacant?

Comment (shorowit) [reply HPXMLtoOpenStudio/resources/data/unavailable_periods.csv:27]: Seems like a reasonable default to me.

**Comment by joseph-robertson** [comment]
> Is there any way to add a test that would proactively identify a missing schedule in the csv?

I think the tests in `test_schedules.rb` take care of that. The problem with recirc pump, however, is that currently those tests operate only on files _without_ recirc pumps.

**Review by shorowit — APPROVED** [review]

**Pull request merged**